Makefile

```diff
@@ -3,7 +3,7 @@ MAKO_FILES = docker-compose.yaml.mako $(shell find .tx doc docker geoportal/test
 VARS_FILE ?= vars.yaml
 VARS_FILES += vars.yaml
 
-DEVELOPPEMENT ?= FALSE
```

Comment:
is this variable still used? `git grep DEVELOPPEMENT` return no results

Reply:
The correct version is steel used :-)